### ISSUE TITLE
Custom Paths

### ISSUE THREAD

**Issue by Christian-Seiler** [openaudible/openaudible#1393]
I'd like to be able to set custom paths for where the audiobooks should be placed.

instead of  `books/<book-title>.m4b` I'd like to have `<author-sort>/<book-title>.m4b`

**Comment by ericthelin** [comment]
having some flexibility in that would be even better as I organize book by `genre/author/seriesName/seriesNum bookTitle/file.m4b` where genre is a relatively short list Fantasy, Science Fiction, Science, History, etc. So matching the genre would be a little tricky but everything else is doable in a simple template and would be huge step forward.

Then again even `author - title.m4b` would be an improvement on what it does now. To me this is probably the weakest thing about the app.

**Comment by openaudible** [comment]
Yes, understand the desire and it is often requested. Have a system in place but we are being very careful about not releasing it too early as any bug could destroy a library's structure and be a giant headache. One of these days.. but no ETA.
